### PR TITLE
fix(config): remove excess hardcoded account

### PR DESCRIPTION
**Pull request by kstdl**
#1525

### CODE REVIEW

**Review by MatusKysel — APPROVED** [review]

File: libraries/config/src/chain_config.cpp [hunk 1]
```diff
@@ -48,12 +48,9 @@ decltype(ChainConfig::predefined_) const ChainConfig::predefined_([] {
     cfg.final_chain.state.eth_chain_config.dao_fork_block = state_api::BlockNumberNIL;
     cfg.final_chain.state.execution_options.disable_nonce_check = true;
     cfg.final_chain.state.execution_options.disable_gas_fee = true;
-    addr_t root_node_addr("de2b1203d72d3549ee2f733b00b2789414c7cea5");
```
Comment (JakubFornadel) [inline libraries/config/src/chain_config.cpp:51]: dont we need to change also genesis dag block sender from few lines above ? `"sender": "de2b1203d72d3549ee2f733b00b2789414c7cea5"`

Comment (kstdl) [reply libraries/config/src/chain_config.cpp:51]: There could be any random address. And it is changing nothing as I see

Comment (JakubFornadel) [reply libraries/config/src/chain_config.cpp:51]: I would say lets remove that particular account from everywhere, where it is not needed so it is not confusing

File: tests/util_test/util.hpp [hunk 2]
```diff
@@ -120,6 +120,10 @@ inline auto make_node_cfgs(uint count) {
       return ret;
     }
     for (auto& cfg : ret) {
+      addr_t root_node_addr("de2b1203d72d3549ee2f733b00b2789414c7cea5");
+      cfg.chain.final_chain.state.genesis_balances[root_node_addr] = 9007199254740991;
+      auto& dpos = *cfg.chain.final_chain.state.dpos;
+      dpos.genesis_state[root_node_addr][root_node_addr] = dpos.eligibility_balance_threshold;
```
Comment (JakubFornadel) [inline tests/util_test/util.hpp:126]: A better place to add this this is probably in chain_config.cpp:
```
cfgs["test"] = [&] {
    auto cfg = cfgs["default"];
    cfg.chain_id = 12345;
    cfg.final_chain.state.genesis_balances[addr_t("de2b1203d72d3549ee2f733b00b2789414c7cea5")] =
        u256(7200999050) * 10000000000000000;  // https://ethereum.stackexchange.com/a/74832
    return cfg;
  }();
```

**Review by mfrankovi — APPROVED** [review]

File: libraries/config/src/chain_config.cpp (hunk 1 above)
Comment (kstdl) [reply libraries/config/src/chain_config.cpp:51]: I replaced it with new random address

**Review by JakubFornadel — APPROVED** [review]

**Pull request merged**